microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Updates to Region information exposed to web content
+description: How customers and web developers can use the OS regional format in Microsoft Edge for improved site experiences
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/15/2022
+--- 
+# Updates to Region information exposed to web content
```

Comment:
```suggestion
# Introducing the OS regional sharing setting
```

Reply:
longer-lasting would be "OS regional sharing".
@JasonAndrewWriter  - see https://developers.google.com/style/timeless-documentation